web/cypress/integration/route/search-route.spec.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* eslint-disable no-undef */
+
+context('Create and Search Route', () => {
+        beforeEach(() => {
+                // init login 
+                cy.login();
+        })
+
+        it('create route test1, test2, test3', () => {
+                //  go to route create page
+                cy.visit('/');
+                cy.contains('Route').click();
+                for (var i = 0; i < 3; i++) {
+                        cy.contains('Create').click();
+                        cy.get('#name').type('test' + i);
+                        cy.get('#desc').type('desc' + i);
+                        cy.get('#hosts_0').type('11.11.11.11');
+                        cy.contains('Next').click();
+                        cy.wait(400);
+                        cy.get('#nodes_0_host').type('12.12.12.12', {
+                                timeout: 500
+                        });
+                        cy.contains('Next').click();
+                        cy.contains('Next').click();
+                        cy.contains('Submit').click();
+                        cy.contains('SubmitSuccessfully');
+                        // back to route list page
+                        cy.contains('Return Route List').click();
+                        cy.url().should('contains', 'routes/list');
```

Comment:
why check this？

Reply:
> why check this？

after click the button with the text  `Return Route List`, we need to ensure that the page jumps to the correct path.